lib/Conversation/CommentForm/CommentFormActions.js

```diff
@@ -30,7 +30,7 @@ const CommentFormActions = props =>
       </Button>
       <Button
         types={['secondary', 'size-sm']}
-        clickHandler={props.onSubmit}
+        clickHandler={() => {}}
         isSubmit
       >
         Send
```

Comment:
Ah, indeed it does! Then it can be removed from the props validation too, that was what initially caused my confusion, it being in two places :)